hyp3_gamma/metadata/templates/insar/unw_phase_png.xml.j2

```diff
@@ -12,13 +12,13 @@ The unwrapped phase browse image is generated from the unwrapped interferogram a
 
 {#
 Begin idAbs READABLE - not to be included in the xml template
-This PNG file is a browse image of the unwrapped interferogram for a pair of Sentinel-1 {{ granule_type }} granules, processed using GAMMA software. The process of unwrapping the wrapped interferogram applies an integer multiple of the 2-pi values sequentially to adjacent fringes, generating the relative phase difference between the two acquisitions across the image. The color fringes indicate phase differences of 6-pi over the extent of one color cycle, which equate to 1.5 times the sensor wavelength in line-of-sight displacement. The unwrapped phase data is underlain with backscatter intensity (amplitude) data to place the phase differences in topographic context. The image is resampled from the unwrapped interferogram to be 2048 pixels wide. This product {{ "had" if water_mask_applied else "did not have" }} a water mask applied during the phase unwrapping process.
+This PNG file is a browse image of the unwrapped interferogram for a pair of Sentinel-1 {{ granule_type }} granules, processed using GAMMA software. The process of unwrapping the wrapped interferogram applies an integer multiple of the 2-pi values sequentially to adjacent fringes, generating the relative phase difference between the two acquisitions across the image. The color fringes indicate phase differences of 6-pi over the extent of one color cycle, which equates to 1.5 times the sensor wavelength in line-of-sight displacement per fringe. The unwrapped phase data is underlain with backscatter intensity (amplitude) data to place the phase differences in topographic context. The image is resampled from the unwrapped interferogram to be 2048 pixels wide. This product {{ "had" if water_mask_applied else "did not have" }} a water mask applied during the phase unwrapping process.
```

Comment:
which is equivalent to 1.5 times the sensor wavelength in line-of-sight displacement.